commit 5af603a441d2cda5e9cd847997954fd378058070 (refs/changes/53/4653/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-08-08T18:20:55+02:00 (1 year, 2 months ago)
    
    TRITON-636 trouble with timeouts in sdcadm health
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/lib/sdcadm.js b/lib/sdcadm.js
index 8506a00..0b09436 100644
--- a/lib/sdcadm.js
+++ b/lib/sdcadm.js
@@ -3936,12 +3936,13 @@ SdcAdm.prototype.checkHealth = function checkHealth(opts, cb) {
         urConnection.exec({
             script: script,
             server_uuid: inst.server,
-            timeout: 5000
+            timeout: 10000
         }, function (err, result) {
             if (err) {
                 inst.health_errors = [ {
                     message: err.message
                 }];
+                self.log.error({error: err}, 'Ur execution error');
                 next(null, inst);
                 return;
             }
